pkg/plugin/plugin.go

```diff
@@ -1,11 +1,70 @@
 package plugin
 
 import (
+	"fmt"
+	"net/http"
 	"os"
 
+	"github.com/gin-gonic/gin"
+
 	"github.com/vincenzopalazzo/cln4go/plugin"
 )
 
+type LOGIN struct {
+	PASSWORD string `json:"password" binding:"required"`
+}
+
+type TLSFiles struct {
+	CA        string `json:"ca"`
+	CLIENTKEY string `json:"client_key"`
+	CLIENT    string `json:"client"`
+}
+
+func readTlsFile() (TLSFiles, error){
+	home, err := os.UserHomeDir()
+	if err != nil {
+	  return TLSFiles{}, err
+	}
+	clientKey, err := os.ReadFile(home + "/.lightning/testnet/client-key.pem")
+	ca, err := os.ReadFile(home + ".lightning/testnet/ca.pem")
+	client, err := os.ReadFile(home + ".lightning/testnet/client.pem")
+	if err != nil{
+	  return TLSFiles{}, err
+	}
+	files := TLSFiles{CA: string(ca), CLIENTKEY: string(clientKey), CLIENT: string(client)}
+	return files, nil
+}
+
+func startServer() {
+	r := gin.Default()
+	r.GET("/ping", func(c *gin.Context) {
```

Comment:
```suggestion
	plugin.server.GET("/ping", func(c *gin.Context) {
```

this for all the instance of r